src/DB/projects.json

```diff
@@ -3361,7 +3361,26 @@
       "tech": ["PYTHON"]
     }
   ]
+ },
+ {
+  "github_username": "Pro-18",
+  "Social_media": {
+    "gitHub": "https://github.com/Pro-18",
+    "LinkedIn": "https://www.linkedin.com/in/harsh-pandey-501903223/",
+    "Instagram": "#",
+    "YouTube": "#",
+    "Twitter": "#"
   },
+  "Projects": [
+    {
+      "link": "https://github.com/Pro-18/wallpaper_viewer",
+      "title": "Wallpaper_Viewer",
+      "description": "The Python Wallpaper_Viewer project is that allows users to view their collection of wallpapers using a GUI built with the Tkinter library",
+      "tech": ["PYTHON"]
+    }
```

Comment:
add the sample code to your previous code you don't need to add full code multiple times